### PR TITLE
feat: updated the docker examples to generate consensus keys and use gossipnet

### PR DESCRIPTION
**Pull request by pompon0**
Tested on a hetzner instance.

### CODE REVIEW

**Comment by brunoffranca** [comment]
If possible, we could enable consensus by default.

**Review by RomanBrodetski — APPROVED** [review]
Approved - but please also wait for @tomg10 's approval

**Review by tomg10 — APPROVED** [review]

**Pull request merged**
